core/graphelier-service/models/playback.go

```diff
@@ -0,0 +1,91 @@
+package models
+
+type Modification interface{}
```

Comment:
All structs will have implemented this interface

Reply:
Interfaces in golang are not explicitly implemented. https://gobyexample.com/interfaces